scripts/irods/test/test_chunkydevtest.py

```diff
@@ -72,7 +72,7 @@ def test_beginning_from_devtest(self):
         self.admin.assert_icommand("ils -A " + irodshome + "/icmdtest/foo1",
                                    'STDOUT_SINGLELINE', testuser1 + "#" + irodszone + ":read")
         # basic replica
-        self.admin.assert_icommand("irepl -B -R " + self.testresc + " --rlock " + irodshome + "/icmdtest/foo1")
```

Comment:
I'm guessing you learned about these through the removal of support for -B?

Reply:
Yep, it's what I talked about in this comment: https://github.com/irods/irods/pull/8340#discussion_r2019328658.

I'm not sure if removing the `-B` leaves the tests in a valid state.

Reply:
Cool cool. I agree with the comments in that thread.